app/javascript/greetings/greetingsSlice.js

```diff
@@ -0,0 +1,36 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+const GREETINGS_URL = "http://127.0.0.1:3000/api/v1/greetings";
```

Comment:
- [ ] Hey there! 🌐 Just a heads-up: using full URLs might trigger cross-site origin issues, leading to blocked requests. 🚫 To avoid this, opt for relative URLs when serving React inside Rails. 🚀 This ensures that the browser treats the requests as originating from the same domain, preventing any hiccups. Happy coding! 👩‍💻👨‍💻

```suggestion
const GREETINGS_URL = "/api/v1/greetings";
```